packages/framework/src/shared/dto.ts

```diff
@@ -34,3 +36,11 @@ export interface ScriptDTO {
   name: string;
   content: string;
 }
+
+export interface EnsembleEnvironmentDTO {
```

Comment:
This should just be a generic Record<string unknown> type since the user can put any environment variables they want.